Core/src/IfcGeometryConverter/RepresentationConverter.h

```diff
@@ -784,6 +786,7 @@ namespace OpenInfraPlatform {
 
 				// std::shared_ptr<StylesConverter>	stylesConverter;
 				std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> placementConverter;
+				std::shared_ptr<SplineConverterT<IfcEntityTypesT>> splineConverter;

```

Comment:
Since it is not needed anywhere in the class, there is no need for a member variable.

Reply:
Done in commit b13e888